packages/manager/src/features/ObjectStorage/BucketDetail/BucketDetail.tsx

```diff
@@ -279,7 +280,13 @@ export const BucketDetail = (props: Props) => {
   };
 
   const addOneFile = (objectName: string, sizeInBytes: number) => {
-    if (!data) {
+    const currentData:
+      | InfiniteData<ObjectStorageObjectList, unknown>
+      | undefined = queryClient.getQueryData(
+      getObjectBucketObjectsQueryKey(clusterId, bucketName, prefix)
+    );
```

Comment:
I think we can accomplish the same thing by setting the generic type of `getQueryData`

```suggestion
    const currentData = queryClient.getQueryData<
      InfiniteData<ObjectStorageObjectList>
    >(getObjectBucketObjectsQueryKey(clusterId, bucketName, prefix));
```

Reply:
Good idea, thank you!